Vienna/Sources/Database/Database.m

```diff
@@ -659,6 +659,33 @@ -(BOOL)setFeedURL:(NSString *)feed_url forFolder:(NSInteger)folderId
 	return YES;
 }
 
+/**
+ *  Sets the identifier of the feed on the remote OpenReader server.
+ *
+ *  @param remoteId the identifier to set the folder's feed to
```

Comment:
Could you please add an example of this identifier? I understand it's the hexadecimal ID but it would be nice to have a bit more detail in the comments for the future.